sdk/python/feast/repo_operations.py

```diff
@@ -116,6 +116,9 @@ def apply_total(repo_config: RepoConfig, repo_path: Path):
 
     os.chdir(repo_path)
     registry_config = repo_config.get_registry_config()
+
+    print(type(registry_config))
```

Comment:
what is this for?

Reply:
ah oops, forgot to remove this when cleaning up. Thanks for catching